src/services/utils.ts

```diff
@@ -64,19 +126,31 @@ const prepareRequest = (init: FetchInitWithTimeout | undefined, token?: string)
 
 const handleError = (response: Response) => {
     return response.text().then((text: string) => {
-        const errorName = 'HttpResponseError : ';
         const errorJson = parseError(text);
-        let customError: Error & { status?: number };
-        if (errorJson && errorJson.status && errorJson.error && errorJson.message) {
-            customError = new Error(
-                `${errorName + errorJson.status} ${errorJson.error}, message : ${errorJson.message}`
-            );
-            customError.status = errorJson.status;
-        } else {
-            customError = new Error(`${errorName + response.status} ${response.statusText}, message : ${text}`);
-            customError.status = response.status;
+        const fallbackStatus = response.status;
```

Comment:
I think you can keep it simpler. if ('businessErrorCode' in errorJson) then throw new CustomError({messageId: errorJson.businessErrorCode}) else throw new CustomError({messageTxt: errorJson.message}).
Something like this.